.github/workflows/ghcr.yml

```diff
@@ -11,6 +11,21 @@ on:
     tags:
       - '*'
   pull_request:
+    paths:
```

Comment:
Do we really need `paths`? It seems what you would like to achieve can be solely done by `paths-ignore`?

Reply:
I'm aiming not to just exclude `docs`, but trying to find out if we can iterate on `paths` to really only include the relevant folders that would justify the extensive image generation?

Reply:
That makes sense. The only concern of mine is that it's much easier to forget to add sth to `paths` list than `paths-ignore` list.